mern-ecommerce/controllers/category.js

```diff
@@ -68,3 +68,36 @@ exports.list = (req, res) => {
     res.json(data);
   });
 };
+
+exports.addProductToCategory = (req, res) => {
+  const { categoryId } = req.params;
+  const { productId } = req.body;
+
+  Category.findById(categoryId, (err, category) => {
```

Comment:
The error handling in this code snippet is consistent with the rest of the repository context, which is a good practice. However, it would be better to use a more descriptive error message instead of just "Category not found". Consider using something like "Category with id ${categoryId} not found" to provide more context.